src/Libraries/Display/Display.cs

```diff
@@ -14,9 +15,11 @@ public class Display :  IGraphicItem
     {
         #region private members
 
+        private readonly Point[] vertices;
```

Comment:
I think potentially exposing properties for some of these might be useful. Imagine a mesh export node that takes a display object and extracts the colors for string vertex color information.